frontend/src/app/projects/project-overview/project-overview.component.html

```diff
@@ -22,7 +22,11 @@
       *ngFor="let project of (projectService.projects$ | async)!"
       [routerLink]="['/project', project.slug]"
     >
-      <mat-card matRipple class="mat-card-overview">
+      <mat-card
+        [ngClass]="{ 'bg-gray-300': project.is_archived }"
```

Comment:
```suggestion
        [ngClass]="{ 'bg-archived': project.is_archived }"
```

Let's add a tailwind variable for it for more consistency.